test/dump/reference-types.txt

```diff
@@ -84,10 +84,12 @@ Table[3]:
  - table[1] type=anyref initial=1
  - table[2] type=funcref initial=1
 Elem[2]:
- - segment[0] flags=2 table=2 count=1 - init i32=0
+ - segment[0] flags=0 table=0 count=1 - init i32=0
```

Comment:
This doesn't look right.  Did something change such that `(elem $baz (i32.const 0) $f1)` not long means that the elements are for table $baz?

Reply:
Oh I guess it pulled in bulk memory and that changes the meaning here?   In which case we now have elem and table with the same name.. is that allowed?

Reply:
Yeah, the names are kept in separate namespaces. We always know what kind of thing we are referring to.

Reply:
Oh yeah, and I guess the new syntax for referring to a table is `(table $baz)`